lib/src/ext/extensions.dart

```diff
@@ -68,3 +61,16 @@ final List<Extension> defaultExtensions = <Extension>[
   new CesiumRtcExtension(),
   new Web3dQuantizedAttributesExtension()
 ];
+
+// https://github.com/KhronosGroup/glTF/blob/master/extensions/Prefixes.md
+const List<String> kReservedPrefixes = const <String>[
+  'KHR_',
```

Comment:
I wonder if we'll need a reminder to update this list from time to time.

Reply:
What would you suggest? Adding a comment here or adding a note to `Prefixes.md`, or both?

Reply:
We've already got the link here in a comment.  I think `Prefixes.md` should get a note (after this is merged) that any changes there should be copied here (or bump the glTF-Validator maintainers to do it).